.travis-ci.sh

```diff
@@ -31,17 +31,17 @@ function setup_arm32_chroot {
 	VERSION=jessie
 	CHROOT_ARCH=armhf
 	# Host dependencies
-	sudo apt-get install -qq -y ${HOST_DEPENDENCIES}
+	sudo apt-get install -qq -y "${HOST_DEPENDENCIES[@]}"
```

Comment:
Yep ... if you look back at the 'commits' in this PR, as was simply quoting, then Travis CI etc would fail. It seemed odd to me as well - yep agree, but it works & makes Codacy happy ... go figure.